src/PIL/Image.py

```diff
@@ -3458,10 +3460,18 @@ def _get_ifd_dict(self, offset):
             return self._fixup_dict(info)
 
     def _get_head(self):
+        version = b"\x2B" if self.bigtiff else b"\x2A"
```

Comment:
\x2B is 43, and \x2A is 42, as per https://www.awaresystems.be/imaging/tiff/bigtiff.html